### PR TITLE
Update code resend button cooldown based on rate limit

### LINKED ISSUE #953: Email verification code expired UX enhancement
- If the code is expired, use the resend button to request a new email
- In the error message, "Resend a new one" should be a link to resend the code
- Then add a button to back to the app login page.
(Pending design)

Input the expired verification code:
image
- Allow resending immediately

### PR DESCRIPTION
**Pull request by carmenlau**
fix #953 

~~WIP: After changing the button state based on rate limit, the resend button can be clicked multiple times without cooldown. So now the resend link will not change to count down after click. We need to add some visual feedback to let the user know the code is resent.~~

Updated using flash message to show "resend code successfully message".

### CODE REVIEW

File: authui/src/authgear.ts [hunk 1]
```diff
@@ -258,6 +258,38 @@ window.api.onLoad(() => {
   }
 });
 
+// Handle click link to submit form
+// When clicking element with `data-submit-link`, it will perform click on
+// element with `data-submit-form` that contains the same value
+// e.g. data-submit-link="verify-identity-resend" and
+//      data-submit-form="verify-identity-resend"
```
Comment (louischan-oursky) [inline authui/src/authgear.ts:265]: This is smart!

**Review by louischan-oursky — COMMENTED** [review]
LGTM! Just some very minor suggestions

**Comment by carmenlau** [comment]
Updated

**Comment by carmenlau** [comment]
Added 2 commits to address the problems we mentioned in the product meeting.
1. Added new rate limit keys listed below, finally i didn't put channel (email/sms) into the key since it is not useful for counting this rate limit.
    - `verification-send-code:{target}`
    - `oob-send-code:setup-primary-oob:{target}`
    - `oob-send-code:setup-secondary-oob:{target}`
    - `oob-send-code:authenticate-primary-oob:{target}`
    - `oob-send-code:authenticate-secondary-oob:{target}`
2. Enforce verification code_expiry_seconds > 60

**Pull request merged**